browser/brave_wallet/BUILD.gn

```diff
@@ -72,7 +72,9 @@ source_set("brave_wallet_delegate") {
     deps += [
       "//brave/browser/brave_wallet",
       "//brave/browser/brave_wallet:tab_helper",
+      "//brave/browser/ui:ui",
```

Comment:
nit: just "//brave/browser/ui", same as below.

Reply:
Fixed and rebased in 10bd683f95363d743b2fb72a09278f0b5c3bb92e